packages/threat-composer/src/customTypes/assumptions.ts

```diff
@@ -17,10 +17,7 @@ import { z } from 'zod';
 import { ContentEntityBaseSchema, EntityLinkBaseSchema } from './entities';
 
 export const AssumptionSchema = ContentEntityBaseSchema.extend({
-  /**
-   * A flag determining the assumption is still valid or not.
-   */
-  valid: z.boolean().optional(),
```

Comment:
Take note please. I did not see any code paths that used this, so I dropped it

Reply:
Probably it is something we are plan to do but have not implemented yet.

Reply:
Ok, I'm going to keep this as removed / deleted from the schema then